doc/game.md

```diff
@@ -1,18 +1,82 @@
-# Game Loop
+# BaseGame
 
-The Game Loop module is a simple abstraction over the game loop concept. Basically most games are built upon two methods:
+`BaseGame` is the most basic and most commonly used `Game` class in Flame.
 
-* The render method takes the canvas ready for drawing the current state of the game.
-* The update method receives the delta time in seconds since last update and allows you to move to the next state.
+The `BaseGame` class implements a `Component` based `Game` for you; basically it has a list of
+`Component`s and passes the `update` and `render` calls to all `Component`s that have been added to
+the game. You can still extend those methods to add custom behavior, and you will get a few other
+features for free, like the passing of `resize` methods (every time the screen is resized the
+information will be passed to the resize methods of all your components) and also a basic camera
+feature.
 
-The `Game` class can be extended and will provide these gameloop methods and then its instance Flutter widget tree via the `GameWidget`.
+The `BaseGame.camera` controls which point in the coordinate space that should be the top-left of
```

Comment:
```suggestion
The `BaseGame.camera` controls which point in the coordinate space should be the top-left of
```

Reply:
Isn't it easier to read with "that" there?

Reply:
Imo I found that it made it harder to read. And context wise it felt it didnt fit.

`Which point in the coordinate space that should`, what does that reference, the point? If so we can remove it because we are still talking about that point.